packages/manager/apps/web/client/app/components/order-contracts/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,7 @@
+{
+  "order_contracts": {
+    "label": "Je reconnais avoir bien pris connaissance",
+    "separator_and": "et",
+    "separator_of": "des"
```

Comment:
Both are not translatable, please do otherwise :)

Reply:
Was done like that on another manager. I'll check another solution.
Have you an idea on your side ?

Reply:
Splitted in 4 more friendly translations